### DISCUSSION TITLE
Unable to sign up for student pack with remote Masters

### DISCUSSION THREAD

**Discussion by Louis-gatech** [community/community#133541] [GitHub Education]
### Hi everyone,

Hi everyone, I am enrolled to Georgia Tech's Online Masters. I am not residing in the US and hence I am required to send a proof that I am a remote student. I have tried to submit a document stating that I am a remote student however it did not work. Anybody successfully managed to get the student benefits with an online master? what did you upload?

**Comment by redbankdev** [comment]
I'm having the same issue. I recently had to contact support because I was unable to upload an unofficial transcript to show proof of enrollment (I'm hoping they finally accept that, at least)

**Comment by johngai19** [comment]
I am taking an online master program of university of pittsburgh from coursera, I have the school email, but I cannot register for the student pack either, because I am not reside in the US.
